boards.txt

```diff
@@ -4954,6 +4954,14 @@ GenL4.menu.pnum.GENERIC_L422KBUX.build.board=GENERIC_L422KBUX
 GenL4.menu.pnum.GENERIC_L422KBUX.build.product_line=STM32L422xx
 GenL4.menu.pnum.GENERIC_L422KBUX.build.variant=STM32L4xx/L412K(8-B)(T-U)_L422KB(T-U)
 
+# Generic L431RCTx
+GenL4.menu.pnum.GENERIC_L431RCTX=Generic L431RCTx
+GenL4.menu.pnum.GENERIC_L431RCTX.upload.maximum_size=262144
+GenL4.menu.pnum.GENERIC_L431RCTX.upload.maximum_data_size=65536
+GenL4.menu.pnum.GENERIC_L431RCTX.build.board=GENERIC_L431RCTX 
+GenL4.menu.pnum.GENERIC_L431RCTX.build.product_line=STM32L431xx
+GenL4.menu.pnum.GENERIC_L431RCTX.build.variant=STM32L4xx/L431R(B-C)(I-T-Y)
+
```

Comment:
Would be great if you could add all the generic MCU you can found in boards_entry.txt, as they share the same configuration files (it is just a copy and past of boards_entry.txt, but don't forget to also add them in README.md)